import "IpcMessage.proto";
import "User.proto";
import "ClassCategory.proto";

package METRO.CRO.MESSAGES;

message UserInit {
	extend IpcMessage {
		optional UserInit message = 109;
	}

	required User userinfo = 1;
	repeated ClassCategory categories = 2;
}